src/pages/driver/luggage-policies/index.tsx

```
@@ -140,76 +139,54 @@ const LuggagePolicyRow = ({
-  const { 
-    data: policies = [], 
-    isLoading, 
-    error 
-  } = useQuery<LuggagePolicy[], Error>({
-    queryKey: ['driver-luggage-policies'],
-    queryFn: getDriverLuggagePolicies,
-    retry: 1
-  });
+  const policies = useDriverLuggagePolicies();
+  const isLoading = policies === undefined;
+  const error = null; // Convex handles errors differently
```

ESLint

[error] 'error' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)